src/styles/style.css

```diff
@@ -1,3 +1,104 @@
 body {
-  background-color: aquamarine;
+  margin: 0;
+  padding: 0;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  background-color: whitesmoke;
+}
+
+#wrapper {
+  padding: 1rem;
+}
+
+#header-title {
+  font-size: 4rem;
+}
+
+.main {
+  display: flex;
+  flex-direction: row;
+  padding: 2rem;
+  gap: 1rem;
+  background-color: #ffcd74d1;
+}
+
+.score-header {
+  display: flex;
```

Comment:
- [optional] Good job implementing the requirements, however, you added styles _that_ are not required, as the requirements needed just a wireframe, Learning to follow requirements strictly is a good skill to have.

Reply:
okay thanks